vcd/provider.go

```diff
@@ -246,6 +246,13 @@ func Provider() *schema.Provider {
 				Description: "The API token used instead of username/password for VCD API operations. (Requires VCD 10.3.1+)",
 			},
 
+			"service_account": {
```

Comment:
Maybe it's too long, but should it be `service_account_token_file`? Or something shorter, but that highlights that it is a file and contains the token.

Reply:
Changed. I don't like that it's very long, but it probably provides much more context to the user.

Reply:
Yeah, let's see what @Didainius and @lvirbalas think about it also.

Reply:
I vote for `service_account_token_file` in this case too. Mainly because `provider` section does not take a lot of space in overall configuration and it's better to be extra clear here.

Reply:
+1 for `service_account_token_file`

Reply:
Changed.